### ISSUE TITLE
[BUG] Json is not generating settings on dev 1.19 branch

### ISSUE THREAD

**Issue by JosephMillsAtWork** [AntMCDev/MCSkyblock#48]
One can reproduce if they remove the json that is under the config dir and start minecraft.  

Issue: json is just a empty object. and is not writing config back and forth anymore.

**Comment by JosephMillsAtWork** [comment]
Comes down to static context. I added a simple 

```java
@Config(name = "mcskyblock" )
public class SkyBlockConfig implements ConfigData {

    public int testInt = 0;
```

and test int came back on the json. 

But 

```java
@Config(name = "mcskyblock" )
public class SkyBlockConfig implements ConfigData {

    public static int testInt = 0;
```

did not

So a Simple fix is as follows., 

1) Create the following in the MCSkyBlock Class

```java
    private static SkyBlockConfig CONFIGS;
    public static SkyBlockConfig.WorldGen CONFIG_GEN;
```

then in the _init._

```java
   public static void init() {
        AutoConfig.register(SkyBlockConfig.class, GsonConfigSerializer::new);
        CONFIGS = AutoConfig.getConfigHolder(SkyBlockConfig.class).getConfig();
        CONFIG_GEN = CONFIGS.worldGen;
```

Then in all places to set or read the data. 

**From**

```
SkyBlockConfig.WorldGen.IS_OVERWORLD_SKYBLOCK
```

**TO**

```
MCSkyBlock.CONFIG_GEN.IS_OVERWORLD_SKYBLOCK
```

Wonky but works. json comes back and settings work again.

**Comment by AntMCDev** [comment]
Fixed on commit e5983849
